docs/install.rst

```diff
@@ -0,0 +1,96 @@
+############
+Installation
+############
+
+This page outlines how to install one of the officially distributed SkyPy
+releases and it's dependencies, or install and test the latest development
+version.
+
+From PyPI
+---------
+
+All  SkyPy releases are distributed through the Python Package Index (PyPI_).
+To install the latest version use pip_:
+
+.. code:: console
+
+    $ pip install skypy
+
+From conda-forge
+----------------
+
+All SkyPy releases are also distributed for conda_ through the `conda-forge`_
+channel. To install the latest version for your active conda environment:
+
+.. code:: console
+
+    $ conda install -c conda-forge skypy
+
+From GitHub
+-----------
+
+The latest development version of SkyPy can be found on the master branch of
+the `skypyproject/skypy`_ GitHub repository. This and any other branch or tag
+can be installed directly from GitHub using pip:
+
+.. code:: console
+
+    $ pip install git+https://github.com/skypyproject/skypy@master
+
+Dependencies
+------------
+
+SkyPy is compatble with Python versions 3.6 or later on Ubuntu, macOS and
+Windows operating systems. It has the following core dependencies:
+
+- `astropy <https://www.astropy.org/>`__
+- `networkx <https://networkx.github.io/>`_
+- `numpy <https://numpy.org/>`_
+- `pyyaml <https://pyyaml.org/>`_
+- `scipy <https://www.scipy.org/>`_
+
+Installing using pip or conda will automatically install or update these core
+dependencies if necessary. SkyPy also has a number of optional dependencies
+that enable additional features:
+
+- `camb <https://camb.info/>`_
+- `classy <https://lesgourg.github.io/class_public/class.html>`_
+- `healpy <https://healpy.readthedocs.io/>`_
+- `speclite <https://speclite.readthedocs.io/>`_
+- `specutils <https://specutils.readthedocs.io/>`_
+
```

Comment:
Do we want to mention that all of these can be installed with `pip install skypy[all]`?